test_scripts/Resumption/InteriorVehicleData/004_InteriorVD_resumption_all_modules_unexpected_disconnect.lua

```diff
@@ -0,0 +1,69 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1. App is subscribed to all modules
+-- 2. Transport disconnect and reconnect are performed
+-- 3. App starts registration with actual hashId after unexpected disconnect
+-- SDL does:
+-- 1. send RC.GetInteriorVD(subscribe=true) to HMI during resumption data for all modules
+-- 2. respond RAI(SUCCESS) to mobile app
+-- 3. update hashId after successful resumption
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Resumption/InteriorVehicleData/common_resumptionsInteriorVD')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function checkResumptionData()
+  local modulesToExpect = common.cloneTable(common.modules)
+  EXPECT_HMICALL("RC.GetInteriorVehicleData")
+  :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS",
+        { moduleData = common.getModuleControlData(data.params.moduleData.moduleType), isSubscribed = true })
+    end)
+  :ValidIf(function(exp, data)
+    modulesToExpect[data.params.moduleData.moduleType] = true
```

Comment:
@GetmanetsIrina since the hmi request does not contain module type, maybe we should try to access `data.params.moduleType`  instead of `data.params.moduleData.moduleType`?